### DISCUSSION TITLE
EGKK southern SIDs

### DISCUSSION THREAD

**Discussion by scotflieger** [VFPC/VFPC#150] [General]
There is a error with the SIDs departures to the south from EGKK. BOGNA1X/M are in use from 26L only from 0600-2300. VFPC is reporting this as incorrect (RST).

SFD1X/5M from 26L is used at night (2300-0600) and SFD4Z/9P from 08R. I have not checked these.

Is this a UK controller pack (latest version used) or VFPC issue?

**Comment by lennycolton** [comment]
The SIDs are coded correctly. Due to #149, VFPC is currently unable to tell what time it is, so it assumes midnight. A fix will be released as soon as possible.

**Comment by lennycolton** [comment]
Fixed in v3.6.4
